tests/test-environments/Jenkinsfile

```diff
@@ -54,11 +54,11 @@ podTemplate(label: label,
                         }
 
                         stage("install dependencies $application") {
-                            execute("make resolve", workDir, componentDir)
+                            execute("dep ensure -vendor-only -v", workDir, componentDir)
                         }
 
                         stage("code quality $application") {
-                            execute("make validate", workDir, componentDir)
+                            execute("gometalinter --vendor --deadline=2m --disable-all --enable=vet", workDir, componentDir)
```

Comment:
I highly recomend creating make commands for this. Thanks to that it will be easy to reproduce what Jenkins is doing locally. Jenkins file should not contain any build logic, it's only orchestrator in my opinion.

Reply:
Or a script, I have no preference. The important point is there is no build logic in Jenkinsfile